code/_onclick/item_attack.dm

```diff
@@ -1,5 +1,5 @@
 /obj/item/proc/melee_attack_chain(mob/user, atom/target, params)
-	if(!tool_attack_chain(user, target) && pre_attackby(target, user, params))
+	if(!tool_attack_chain(user, target) && !surgery_attack_chain(user, target) && pre_attackby(target, user, params))
```

Comment:
I've not looked at any of the rest of the code, but won't the order of this line interfere with things like welding tools, which can be used both as a tool and as a surgery tool?

You might want to stick the surgery attack chain first if it does

Reply:
It will interfere if the target (the mob in case of surgery) also has a tool_act. I hope this isn't ever the case though :P

Reply:
Changed it just in case. Thanks for the tip!